package.json

```diff
@@ -16,6 +16,8 @@
   "dependencies": {
     "@types/express": "^4.17.14",
     "@types/node": "^18.11.18",
+    "classname": "^0.0.0",
+    "classnames": "^2.5.1",
```

Comment:
I think these may be accidental installs in the server package.json? if you cd into the root of the project and run npm uninstall classname, npm uninstall classnames, and npm uninstall sass that should remove them.